css/index.css

```diff
@@ -164,12 +169,67 @@ header .logo {
 /* Contact Styles */
 
 .contact p {
+    font-size: 1.6rem;
     margin: 15px 0;
 }
 
 /* Footer Styles */
 
 footer {
+    font-size: 1.6rem;
     text-align: center;
     margin: 40px 0 20px;
-}
\ No newline at end of file
+}
+
+/*Tablet*/
+
+@media(max-width:768px){
+    .container{
+        width:100%;
+    }
+header .logo{
+    
```

Comment:
if you implement a percentage-based width on the logo image, it will smoothly resize as the viewport size changes, which is great for transitioning between one break point and the next.